### PR TITLE
submitting project

### PR DESCRIPTION
**Pull request by ghost**
# Hotel
Congratulations! You're submitting your assignment!

## Comprehension Questions
Question | Answer
:------------- | :-------------
Describe a design decision you had to make when working on this project. What options were you considering? What helped you make your final decision? | 
Describe a concept that you gained more clarity on as you worked on this assignment. | 
Describe a _nominal test_ that you wrote for this assignment. | 
Describe an _edge case test_ that you wrote for this assignment. | 
How do you feel you did in writing pseudocode first, then writing the tests and then the code? |

### CODE REVIEW

File: lib/availability.rb [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      invalid?(start_date, end_date)
+      available_rooms = @rooms_reservations
+      # change, all_rooms - hash
+
+
+
+    end
+
+    def book_room(start_date, end_date)
+      available_rooms = find_room(start_date, end_date)
```
Comment (kariabancroft) [inline lib/availability.rb:45]: Where is `find_room` implemented?

File: lib/availability.rb [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+      return new_reservation
+    end
+
+    def book_room
+    end
+
+    def available_rooms(start_date, end_date)
+      invalid?(start_date, end_date)
+      available_rooms = @rooms_reservations
```
Comment (kariabancroft) [inline lib/availability.rb:37]: Where is this instance variable set?

File: lib/room.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+module Hotel
+  class Room
+
+    def initialize(room_number)
+      @room_number = room_number
+
+    end
+
+    def room_number
```
Comment (kariabancroft) [inline lib/room.rb:11]: Why would this be a method and not an `attr_accessor`?

File: lib/booking.rb [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+    def initialize(start_date, end_date)
+      @start_date = start_date
+      @end_date = end_date
+      @room = room
+
+    end
+
+    def start_date
+      return @start_date.strftime
```
Comment (kariabancroft) [inline lib/booking.rb:15]: This logic should be done in the initialize and then just use `attr_accessor`s to access the updated data

**Comment by kariabancroft** [comment]
# Hotel
## What We're Looking For

Feature | Feedback
---     | ---
**Design** |
Demonstrated classes having a single responsibility | Yes, mostly. Though a few of the basics usages of instance variables are misguided. 
Demonstrated loose coupling | Yes
Methods demonstrate a good use of encapsulation, inputs and outputs | Yes
Wave 1 requirements | Incomplete
Wave 2 requirements | Incomplete
Wave 3 requirements | No

You absolutely need to be using git more often. This is non-negotiable. You also must be answering the comprehension questions when you turn in your assignments. It seems that much of this submitted code would not run, as-is, so you need you get this code working and notify your instructor when this is done.
